source/includes/localhost-connection.rst

```diff
@@ -1,10 +1,26 @@
-To connect to a database running locally on its default port, change the
-connection string to ``"mongodb://localhost"``. 
+If you need to run a MongoDB server on your local machine for development
+purposes instead of using an Atlas cluster, you need to complete the following:
 
-To connect to a database running locally, change the connection string
-to ``"mongodb://localhost:<port>"``.
+1. Download the `Community <https://www.mongodb.com/try/download/community>`__
+   or `Enterprise <https://www.mongodb.com/try/download/enterprise>`__ version
+   of MongoDB Server.
 
-Your ``mongod`` instance must be running to successfully connect to your
-database. For information on how to start your ``mongod`` instance,
-see the :manual:`Manage mongod Processes
-</tutorial/manage-mongodb-processes/#start-mongod-processes>` Server Manual Entry.
+#. `Install and configure <https://docs.mongodb.com/manual/installation/>`__
+   MongoDB Server.
+
+#. Start the server.
+
+.. warning::
+
+   Never run a MongoDB server unless you have appropriate security measures
```

Comment:
suggestion: I think this should clarify the context where the warning applies. Perhaps something like

>  Never run and expose a MongoDB server to incoming network connections unless you have ...

Reply:
I had something like this before, but decided to delegate security recommendations to the linked page since there may be too many situations/considerations to list. For the Jasmin persona, it also might not be immediately apparent what it means to expose a server to incoming network connections. 
If including the link to serve this purpose doesn't work for you, can you provide the exact information want to include? Thanks.

Reply:
As is, it reads as bordering on alarmist. I grant that it may be because I'm aware of the security implications and Jasmin may not be.

Reply:
> As is, it reads as bordering on alarmist. I grant that it may be because I'm aware of the security implications and Jasmin may not be.

I see, is this a different suggestion? Could you let me know what you'd like to see here instead?

Reply:
I think my concern is with the wording "Never run...". Maybe just "Never run a MongoDB server in production". Or,

> Only run MongoDB locally for testing purposes. When running MongoDB in production, ensure you follow the appropriate security measures...

Something like that.